code/modules/admin/admin.dm

```diff
@@ -704,7 +704,7 @@ var/global/noir = 0
 			if (src.level >= LEVEL_SA)
 				var/M = href_list["target"]
 				var/job = href_list["type"]
-				var/list/cache = apiHandler.queryAPI("jobbans/get/player", list("ckey"=M), 1)[M]
+				var/list/cache = jobban_get_for_player(M)
```

Comment:
usage does not match definition, i assume M is a mob not a ckey

Reply:
It's not, it's a ckey apparently

Reply:
please change it from M then

Reply:
No, out of scope